zstacklib/zstacklib/network/ovs/dpdk.py

```
@@ -0,0 +1,626 @@
+        count = 0
+        for pci in slaves:
+            pf_name += f'{bond.name}_pf{count} '
+            ifce_set += f'-- set Interface {bond.name}_pf{count} type=dpdk options:dpdk-devargs={pci} '
+            count += 1
+
+        cmd += pf_name
+        cmd += f'bond_mode={bond_mode} '
+        if bond_mode == 'balance-tcp':
+            lacp = str(bond.lacp)
+            if lacp not in _VALID_LACP_VALUES:
+                raise OvsError(f'Invalid LACP value: {lacp!r}')
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)